integration_tests/e2e/goal/createGoals.cy.ts

```diff
@@ -57,8 +57,8 @@ context('Create goals', () => {
     Page.verifyOnPage(CreateGoalsPage)
     createGoalPage //
       .hasErrorCount(2)
-      .hasFieldInError('goals[0].title')
-      .hasFieldInError('goals[0].steps[0].title')
+      .hasFieldInError('goals-0-title')
```

Comment:
Because these ids are now automatically created it's easier to implement a more consistent naming convention.